### PR TITLE
[API] Added deprecation page to API tile content

### PR DESCRIPTION
**Pull request by crwaters16**

### CODE REVIEW

File: content/api/reference/deprecations.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+The following changes were made in the new routes:
+
+- `virtual_dns_ips` has been renamed to `dns_firewall_ips`.
+- A disabled ratelimit is now represented by `null` instead of `0`.
+- The `PATCH` method must be used instead of `PUT` to update clusters.
+
+## Zone Analytics API
+**End of life date: March 1, 2021**
+
+This API is deprecated in favor of the (GraphQL Analytics API([https://developers.cloudflare.com/analytics/graphql-api/]), which provides equivalent data and more features, including the ability to select only the metrics that you need. For more information, refer to the (Zone analytics to GraphQL analytics migration guide)[https://developers.cloudflare.com/analytics/graphql-api/migration-guides/zone-analytics/].
```
Comment (dcpena) [inline content/api/reference/deprecations.md]:
```suggestion
This API is deprecated in favor of the [GraphQL Analytics AP](https://developers.cloudflare.com/analytics/graphql-api/), which provides equivalent data and more features, including the ability to select only the metrics that you need. For more information, refer to the [Zone analytics to GraphQL analytics migration guide](https://developers.cloudflare.com/analytics/graphql-api/migration-guides/zone-analytics/).
```

File: content/api/reference/deprecations.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+- GET zones/:zone_identifier/analytics/dashboard
+- GET zones/:zone_identifier/analytics/colos
+
+Replacement:
+GraphQL Analytics API
+
+## Zone settings - __cfduid
+**End of life date: May 10, 2021**
+
+The `__cfduid` cookie was set on Cloudflare HTTP responses and used for providing critical performance and security services on behalf of our customers. For more information on why we transitioned away from this cookie, refer to (Deprecating the `__cfduid` cookie)[https://blog.cloudflare.com/deprecating-cfduid-cookie/].
```
Comment (dcpena) [inline content/api/reference/deprecations.md]:
```suggestion
The `__cfduid` cookie was set on Cloudflare HTTP responses and used for providing critical performance and security services on behalf of our customers. For more information on why we transitioned away from this cookie, refer to [Deprecating the `__cfduid` cookie](https://blog.cloudflare.com/deprecating-cfduid-cookie/).
```

**Comment by dcpena** [comment]
Also, where possible, should we include links to the replacement APIs? I know sometimes direct linking is tricky with the API docs.

**Review by dcpena — APPROVED** [review]

**Comment by crwaters16** [comment]
> 

yes - going to note that for a future edit when that's easier to implement

**Pull request merged**